rave-compiler/src/main/java/com/uber/rave/compiler/AnnotationWriter.java

```diff
@@ -182,16 +191,17 @@ void write(@Nullable FloatRange floatRange) {
 
     private void checkAnnotationNotNull(@Nullable Annotation annotation) {
         if (annotation == null) {
-            throw new RuntimeException("For method " + getter.name + " annotation is empty");
+            throw new RuntimeException("For method " + elementName + " annotation is empty");
         }
     }
 
     /**
      * Adds the ignore if statement check to the method and then the check statement itself.
      */
     private void buildStatements(String statementFormat, Object... objects) {
+        String elementPostFix = writeType == WriteType.METHOD ? "()" : "";
```

Comment:
Can we roll the representation for fieldsName/methodName into a generic POJO? We could then capture details like the post fix inside said POJO which would prevent them from leaking (and possibly repeating) in code generation.